packages/core/src/actions/send.ts

```diff
@@ -99,26 +99,41 @@ export function send<
           : params.delay;
       }
 
-      let resolvedTarget = isFunction(params.to)
+      const resolvedTarget = isFunction(params.to)
         ? params.to(ctx, _event.data, meta)
         : params.to;
-      resolvedTarget =
-        isString(resolvedTarget) &&
-        resolvedTarget !== SpecialTargets.Parent &&
-        resolvedTarget !== SpecialTargets.Internal &&
-        resolvedTarget.startsWith('#_')
-          ? resolvedTarget.slice(2)
-          : resolvedTarget;
+      let targetActorRef: AnyActorRef | undefined;
+
+      if (typeof resolvedTarget === 'string') {
+        if (resolvedTarget === SpecialTargets.Parent) {
+          targetActorRef = actorContext?.self._parent;
+        } else if (resolvedTarget === SpecialTargets.Internal) {
+          targetActorRef = actorContext?.self;
+        } else if (resolvedTarget.startsWith('#_')) {
+          // SCXML compatibility
```

Comment:
nit: personally I'd love a direct link in the comment pointing to the SCXML spec we're adhering too

Reply:
Done in [9e3ef29](https://github.com/statelyai/xstate/pull/3455/commits/9e3ef29e2ba20a35cd73014798119ab4e1d7c249)